crates/forge-runner/src/cairo_runner/sierra_casm_runner.rs

```diff
@@ -50,9 +53,17 @@ pub struct SierraCasmRunner {
     /// The casm program matching the Sierra code.
     pub casm_program: CairoProgram,
     #[allow(dead_code)]
-    // Mapping from class_hash to contract info.
+    /// Mapping from class_hash to contract info.
     pub starknet_contracts_info: OrderedHashMap<Felt252, ContractInfo>,
 }
+
+#[allow(dead_code)]
```

Comment:
IMO it would be better to use Option instead.

Reply:
If function can panic, the result value is wrapped in the panic wrapper, so we have to know the size of an inner type. I could just take this size as `Option<i16>`, but I though it would be less obvious to the caller what happens if it's passed or not.